liblxqt-config-cursor/xcr/xcrimg.cpp

```diff
@@ -59,9 +59,9 @@ void XCursorImage::convertARGB2PreMul (QImage &img) {
   switch (img.format()) {
     case QImage::Format_ARGB32_Premultiplied: return;
     case QImage::Format_ARGB32: break;
-    default: img.convertToFormat(QImage::Format_ARGB32/*_Premultiplied*/);
+    default: (void) img.convertToFormat(QImage::Format_ARGB32/*_Premultiplied*/);
```

Comment:
@palinek I understood. With g++ v6.2 and clang v3.9 we don't get the -Wold-style-cast.
`(void)(...)` is the only case that doesn't trigger the warning.

The standard says:
> Any expression can be explicitly converted to type “cv void.” The expression value is discarded.

The cast isn't really a cast. Is more of a discard.